server/app/models/Version.java

```diff
@@ -171,4 +179,29 @@ public boolean removeTombstoneForQuestion(Question question) {
     }
     return this.tombstonedQuestionNames.remove(question.getQuestionDefinition().getName());
   }
+
+  @VisibleForTesting
+  public boolean addTombstoneForProgramForTest(Program program) {
```

Comment:
suggestion (blocking): since this is public, it doesn't need the testing annotation

Reply:
Sounds good. I kept the "ForTest" suffix in the method name. We can remove this if we ever find a need for this to be used outside of testing.